extensions/Push/lib/Serialize.pm

```diff
@@ -11,13 +11,17 @@ use 5.10.1;
 use strict;
 use warnings;
 
+use Bugzilla::Bug;
 use Bugzilla::Constants;
 use Bugzilla::Extension::Push::Util;
 use Bugzilla::Version;
 
 use Scalar::Util 'blessed';
 use JSON ();
 
+my %translated_field_names = reverse %{Bugzilla::Bug::FIELD_MAP()};
```

Comment:
`reverse` isn't necessary here

Reply:
It is necessary for outgoing data. For incoming data you would not need to reverse which is true.